scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -367,7 +367,13 @@ object Type {
     checkParent(ParentChecks.TypeVar)
   }
 
-  @ast class TypedParam(name: Name, typ: Type) extends Type with Member.Type
+  @ast class TypedParam(name: Name, typ: Type, @newField("4.7.8") mods: List[Mod] = Nil)
+      extends Type with Member.Type with Stat.WithMods
+
+  @ast class FuncParam(mods: List[Mod], tpe: Type) extends Type with Stat.WithMods {
```

Comment:
```suggestion
  @ast class FunctionParam(mods: List[Mod], tpe: Type) extends Type with Stat.WithMods {
```
it's easier to read full names I think

Reply:
done